src/Type/Generic/TemplateTypeMap.php

```diff
@@ -17,7 +17,7 @@ public function __construct(array $types)
 		$this->types = $types;
 	}
 
-	public static function empty(): self
+	public static function createEmtpy(): self
```

Comment:
typo :) BTW we could also cache the empty instance

Reply:
arf /o\